app/Tickets/Flipside/class.FlipsideTicketRequestEmail.php

```diff
@@ -41,7 +41,7 @@ public function getHTMLBody()
                 Reminder:  The dates for mailing your ticket request envelope are '.$this->settings['mail_start_date'].' through '
                 .$this->settings['request_stop_date'].'. Your postmark must be within that date range.<br><br>
                 If you need to update your ticket registration before mailing in your ticket request, return to 
-                <a href="https://secure.burningflipside.com/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
+                <a href="'.$this->secureUrl.'/tickets/">secure.burningflipside.com/tickets/</a>.<br><br>
```

Comment:
My issue is about the framework knowing about every site. So currently your pull added: /** The sites URLs */
 +    protected $wwwUrl;
 +    protected $wikiUrl;
 +    protected $profilesUrl;
 +    protected $secureUrl;

To email. If we deprecate the wiki we need to go yank wiki out of there. If we add a site called say "fred" we need to go add fred there.

The difference between this an #19 is that #19 is only grabbing the data it needs about its own site. While your proposed change grabs every site on every other site. It's not needed and adds extra complexity when/if we change the site structure.